packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-out-page/android/signout.md

```diff
@@ -6,10 +6,11 @@ If the user is signed in using the browser-initiated authorization flow, then lo
 
 ### Clear Browser Session
```

Comment:
Can you move this to the Sign Out Guide?: https://developer.okta.com/guides/sign-users-out/aspnet/before-you-begin/